iina/MainWindowController.swift

```diff
@@ -17,6 +17,8 @@ fileprivate let TitleBarHeightWithOSCInFullScreen: CGFloat = 24 + 10
 fileprivate let OSCTopMainViewMarginTop: CGFloat = 26
 fileprivate let OSCTopMainViewMarginTopInFullScreen: CGFloat = 6
 
+fileprivate let PlaylistMinWidth: CGFloat = 240
+fileprivate let PlaylistMaxWidth: CGFloat = 400
```

Comment:
I'm not sure why the max value is set to 400? Is there a concern users will make it too wide, and if so what is the issue if say the maximum width were set to 80% of window width?

Ironically, if I shrink the window the max width allows me to show the playlist covering up to 90% of the window.
(note: my default viewing size is roughly 1600x800 within a 2560x1440)

Is there a reason the max width is kept to such a small amount?

I'll probably just fork the project and make use my own build version with a larger max width.